lesson-02/task-02.js

```diff
@@ -0,0 +1,11 @@
+const calculateEngravingoneWordPrice = function (message, pricePerWord) {
+  let messageArray = message.trim().split(" ");
+  return messageArray.length * pricePerWord;
```

Comment:
У класу String теж є властивість **length**
```suggestion
  return message.length * pricePerWord;
```